### ISSUE TITLE
mkdirp invalid options argument when using recorder on Windows

### ISSUE THREAD

**Issue by joebowbeer** [fand/veda#246]
Env: Atom 1.50.0, veda 2.14.2, Windows 10

The recording appears to start, and the output file path is displayed (blue) after the recording is stopped, but the green alert is never posted and the recording is never saved.

Stack trace shows that mkdirp is failing in the first call to create recordDir:

```
Uncaught (in promise) TypeError: invalid options argument
    at optsArg (.atom\packages\veda\node_modules\mkdirp\lib\opts-arg.js:13:11)
    at mkdirp (.atom\packages\veda\node_modules\mkdirp\index.js:11:10)
    at .atom\packages\veda\node_modules\pify\index.js:33:11
    at new Promise (<anonymous>)
    at .atom\packages\veda\node_modules\pify\index.js:6:9
    at Object.apply (.atom\packages\veda\node_modules\pify\index.js:85:19)
    at Recorder.<anonymous> (.atom\packages\veda\lib\recorder.js:48:28)
```

**Comment by averagehat** [comment]
I get this on MacOS as well, also version 2.14.2

**Comment by fand** [comment]
Thanks @joebowbeer , @averagehat !

The problem was caused by an update of the library that VEDA uses for creating directories.
The bug is fixed in v2.14.3 which is just released. Please try this!
